windows/QMK Toolbox/MainWindow.cs

```diff
@@ -235,7 +235,7 @@ private void LoadKeyboardList()
             {
                 using (var wc = new WebClient())
                 {
-                    var json = wc.DownloadString("https://compile.qmk.fm/v1/keyboards");
+                    var json = wc.DownloadString("http://api.qmk.fm/v1/keyboards");

```

Comment:
Why is this HTTP now?

Reply:
because that's what skully said/linked

Reply:
It's so we don't trigger the cert issues, due to the super old version of .NET